src/components/Icons/icons.js

```diff
@@ -0,0 +1,16 @@
+const icons = {
+  "chevron-down": [
+    "M0 307.2c0-6.552 2.499-13.102 7.499-18.101 9.997-9.998 26.206-9.998 36.203 0l442.698 442.698 442.699-442.698c9.997-9.998 26.206-9.998 36.203 0s9.998 26.206 0 36.203l-460.8 460.8c-9.997 9.998-26.206 9.998-36.203 0l-460.8-460.8c-5-5-7.499-11.55-7.499-18.102z"
+  ],
+  "chevron-up": [
```

Comment:
for these chevrons -- I wonder if we can use one and rotate it using `transform: rotate(...)`